packages/bvaughn-architecture-demo/components/console/Input.tsx

```diff
@@ -43,6 +32,27 @@ export default function Input() {
     searchStateVisibleRef.current = searchState.visible;
   }, [searchState.visible]);
 
+  // If we are not currently paused at an explicit point, find the nearest point and pause there.
+  let pauseId = explicitPauseId;
+  let frameId: FrameId | null = null;
+  if (pauseId === null) {
+    const isLoaded =
+      loadedRegions !== null &&
+      executionPoint !== null &&
+      isPointInRegions(executionPoint, loadedRegions.loaded);
```

Comment:
Conceptually, we need to avoid making API requests outside of loaded regions to avoid `CommandErrors`. I don't know if a better way to do that than this approach (subscribing to the loaded regions and checking to see if the request is within them).

**Unfortunately** time is fuzzier than execution point, and it's possible that we _think_ we're inside of a loaded region when we actually aren't.

I don't know how to work around this at the moment.